shared/src/business/useCases/trialSessions/generatePrintableTrialSessionCopyReportInteractor.ts

```diff
@@ -45,6 +46,12 @@ export const generatePrintableTrialSessionCopyReportInteractor = async (
     throw new UnauthorizedError('Unauthorized');
   }
 
+  const updatedTrialSessionTypes = await applicationContext
```

Comment:
🔧 This doesn't necessarily indicate to me that it's a boolean value. Could it be renamed to express that a little more clearly? For example, `isTrialSessionFilterFeatureEnabled` (not a great name, just an example!)

Reply:
Good point, I went with `areUpdatedTrialSessionTypesEnabled`?